src/registry/index.ts

```diff
@@ -0,0 +1,50 @@
+import { FhirVersion, CapabilityRegistry, ResourceCapabilityStatement } from 'fhir-works-on-aws-interface';
+import { FhirStructureDefinition } from "../implementationGuides";
+
+/**
+ * This class is the single authority over the supported FHIR StructuredDefinition and their definitions
+ */
+export class FHIRStructureDefinitionRegistry implements CapabilityRegistry {
+    private readonly capabilityStatement: ResourceCapabilityStatement;
+
+    constructor(fhirVersion: FhirVersion, compiledImplementationGuides?: any[]) {
```

Comment:
looks like fhirVersion is unused and can be removed

Reply:
Do we want to process and compile FHIR US CORE R3 & R4 as part of the default packaging? Similar to how it's done for SearchParameters? If so, then we could make use of the `fhirVersion` to implement the conditional loading of those default compiled files.

Reply:
We briefly considered that, but we decided that it is not a feature that we want to support right now.

It could simplify the setup for some customers, but we are not sure if the majority will find it useful. The thing is that for a given FHIR version there may be multiple versions of an IG. i.e. for FHIR 4.0.1 there is at least US Core 3.1.0 and 3.1.1 and we are not sure if everyone just want the latest version or if they prefer to use specific versions.

Also, we can't load US Core by default since not all customers want to use it. We'd need some kind of change to `FhirConfig` to specify which of the "precompiled" IGs should be loaded.

It is something that we could do on the future though.

Reply:
Got it! I'll remove the `fhirVersion` param then.

Reply:
Removed the `fhirUser` param.